modules/react/checkbox/lib/Checkbox.tsx

```diff
@@ -87,16 +91,20 @@ const CheckboxInputWrapper = styled('div')<Pick<CheckboxProps, 'disabled'>>({
   alignSelf: 'flex-start',
 });
 
-const CheckboxRipple = styled('span')<Pick<CheckboxProps, 'disabled'>>({
-  borderRadius: borderRadius.circle,
-  boxShadow: `0 0 0 0 ${colors.soap200}`,
-  height: checkboxHeight,
-  transition: 'box-shadow 150ms ease-out',
-  width: checkboxWidth,
-  position: 'absolute',
-  pointerEvents: 'none', // This is a decorative element we don't want it to block clicks to input
-  zIndex: -1,
```

Comment:
I'm not quite sure why we had a negative index on this, so i removed it, maybe someone else has some context here but if the checkbox sits on another element, the ripple wouldn't show up